src/components/dictionaryConcepts/containers/DictionaryConcepts.jsx

```diff
@@ -101,6 +101,7 @@ export class DictionaryConcepts extends Component {
         this.state.type,
         this.state.typeName,
         this.state.collectionName,
+        this.state.searchInput,
```

Comment:
It'll be cool to destructure this.state here. 
like
`const { type, typeName, collectionName, searchInput } = this.state`

Reply:
I would also advise the same.